app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -10,7 +10,7 @@ module HostManagedExtensions
         alias_method_chain :set_hostgroup_defaults, :katello_attributes
         alias_method_chain :info, :katello
         alias_method_chain :smart_proxy_ids, :katello
-
+        belongs_to :kickstart_repository, :class_name => "::Katello::Repository", :foreign_key => :kickstart_repository_id
```

Comment:
I'm fine with those changes, if brad doesn't mind re-testing with those changes i'd be okay with them.